gallery/src/components/NxTreeViewMultiSelect/NxTreeViewMultiSelectPage.tsx

```diff
@@ -5,228 +5,24 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { NxTable, NxP, NxCode, NxList } from '@sonatype/react-shared-components';
+import { NxP, NxCode, NxTextLink, NxWarningAlert } from '@sonatype/react-shared-components';
 
-import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
-
-import NxTreeViewMultiSelectExample from './NxTreeViewMultiSelectExample';
-import NxTreeViewMultiSelectCustomTooltipExample from './NxTreeViewMultiSelectCustomTooltipExample';
-import NxTreeViewMultiSelectDisabledExample from './NxTreeViewMultiSelectDisabledExample';
-
-const nxTreeViewMultiSelectExampleCode = require('./NxTreeViewMultiSelectExample?raw'),
-    nxTreeViewMultiSelectDisabledExampleCode = require('./NxTreeViewMultiSelectDisabledExample?raw'),
-    nxTreeViewMultiSelectCustomTooltipExampleCode =
-        require('./NxTreeViewMultiSelectCustomTooltipExample?raw');
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
 
 const NxTreeViewMultiSelectPage = () =>
   <>
     <GalleryDescriptionTile>
+      <NxWarningAlert>
+        <NxCode>NxTreeViewMultiSelect</NxCode> has been renamed to <NxCode>NxCollapsibleItemsMultiSelect</NxCode>
+        and is now deprecated and will be removed in the future.
+        {/* eslint-disable-next-line max-len */}
+        Please use <NxTextLink href="#/pages/NxCollapsibleItemsMultiSelect">NxCollapsibleItemsMultiSelect</NxTextLink> instead.
```

Comment:
Couldn't you just add decent line breaks here instead of the eslint override?

Reply:
> Couldn't you just add decent line breaks here instead of the eslint override?

Updated:
dfc4d3d4a764e73148f71ff5c7851d8916c2f6d6